src/DssSpell.sol

```diff
@@ -53,23 +48,11 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     //    https://ipfs.io/ipfs/QmPgPVrVxDCGyNR5rGp9JC5AUxppLzUAqvncRJDcxQnX1u
     //
 
-    // Turn office hours off
     function officeHours() public override returns (bool) {
-        return false;
+        return true;
```

Comment:
You can yank this entire function and use the default to `true`

Reply:
I like explicitness, and also, not removing it makes for smaller diffs in the future